Library/Homebrew/formula.rb

```diff
@@ -1381,12 +1381,15 @@ def stage
       mkdir_p env_home
 
       old_home, ENV["HOME"] = ENV["HOME"], env_home
+      old_java_options = ENV["_JAVA_OPTIONS"]
+      ENV.append "_JAVA_OPTIONS", "-Duser.home=#{env_home}"
```

Comment:
Don't know if we want to `prepend` here in case the user has the same local variable set?